app/assets/stylesheets/application.scss

```diff
@@ -22,12 +22,32 @@ $cadetblue: #49aaa5;
 $mediumaquamarine: #7accc8;
 $indianred: #e45e56;
 
-body {
-  background-image: image_url("");
-  background-position: center top;
-  background-position: 0 45px;
-  background-repeat: no-repeat;
-  background-size: cover;
+.welcome {
+  border: 2px solid $whitesmoke;
+  margin-top: 150px;
+  background:rgba(255, 255, 255, 0.5);
+  height: 300px;
+}
+
+.welcome img {
+  height: 150px
+}
+.trip-main-bar {
+  height: 525px;
+}
+
+.trip-main-bar .image_div {
```

Comment:
Merge rule `.trip-main-bar .image_div` with rule on line 35<br>Selector `image_div` should be written in lowercase with hyphens